pkg/asset/k8s.go

```diff
@@ -163,7 +163,7 @@ func secretFromAssets(name, namespace string, assetNames []string, assets Assets
 	})
 }
 
-func mustCreateAssetFromTemplate(name string, template []byte, data interface{}) Asset {
+func MustCreateAssetFromTemplate(name string, template []byte, data interface{}) Asset {
```

Comment:
> bootkube recovery tool 

One difference is that we load the actual manifest bytes from etcd directly. We do not try to generate manifest bytes from a template. So the current recovery tool does not need to public `MustCreateAssetFromTemplate` func.